### ISSUE TITLE
[Tooltip] Issue

### ISSUE THREAD

**Issue by thecristen** [radix-ui/themes#10]
[The example in the documentation](https://www.radix-ui.com/themes/docs/components/tooltip) doesn't work on click, but works on keyboard focus.

I tried this in Safari, Chrome, and Firefox on MacOS Ventura 13.5.

**Comment by joaom00** [comment]
That's the expected behavior see https://github.com/radix-ui/primitives/issues/955#issuecomment-960610209

**Comment by thecristen** [comment]
Ah makes sense, I was conflating the expected behavior with that of a popover. Thanks!

HOWEVER (sorry for not mentioning in my original post), it _also_ doesn't respond to mouse hover either. Wouldn't this be a problem?

From [the ARIA Authoring Practices guide](https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/): 
> A tooltip is a popup that displays information related to an element when the element receives keyboard focus **or the mouse hovers over it**

**Comment by joaom00** [comment]
> HOWEVER (sorry for not mentioning in my original post), it also doesn't respond to mouse hover either. Wouldn't this be a problem?

Hm, seems to work here. There is a slight delay to open the tooltip, so keep your mouse over long enough

**Comment by thecristen** [comment]
Got it!
I think what was happening is that my clicking wasn't merely ignored - the clicking was actively blocking the hover from being understood. Verrryyy interesting. Thanks!
Closing since this is more user error than anything, though I think it'd be a nice improvement if clicks could somehow be ignored without it preventing the hovering from working.
